src/lib.rs

```diff
@@ -1149,6 +1149,131 @@ mod tests {
         });
     }
 
+    #[test]
+    fn should_be_able_to_cancel_only_once() {
+        new_test_ext().execute_with(|| {
+            let multi_id = MultiAccount::multi_account_id(2);
+
+            assert_ok!(MultiAccount::create(Origin::signed(1), 3, vec![2, 3]));
+
+            let call = Box::new(Call::Balances(BalancesCall::transfer(6, 15)));
+            let hash = call.using_encoded(blake2_256);
+            assert_ok!(MultiAccount::approve(
+                Origin::signed(1),
+                multi_id,
+                None,
+                hash.clone()
+            ));
+            assert_ok!(MultiAccount::approve(
+                Origin::signed(2),
+                multi_id,
+                Some(now()),
+                hash.clone()
+            ));
```

Comment:
This second approve is not necessary for the test

Reply:
@ivanceras could you fix this?